Modules/posixmodule.c

```diff
@@ -12607,6 +12607,8 @@ os_cpu_count_impl(PyObject *module)
     ncpu = mpctl(MPC_GETNUMSPUS, NULL, NULL);
 #elif defined(HAVE_SYSCONF) && defined(_SC_NPROCESSORS_ONLN)
     ncpu = sysconf(_SC_NPROCESSORS_ONLN);
+#elif defined(__VXWORKS__)
+    ncpu = __builtin_popcount(vxCpuEnabledGet());
```

Comment:
```suggestion
    ncpu = _Py_popcount32(vxCpuEnabledGet());
```

You may need to add `#include "pycore_bitutils.h"` also.

Reply:
@corona10 Thanks for your review. Have replaced.